extensions/ql-vscode/src/view/model-alerts/ModelAlertsResults.tsx

```diff
@@ -89,3 +101,10 @@ export const ModelAlertsResults = ({
     </div>
   );
 };
+
+function sendRevealInModelEditorMessage(modeledMethod: ModeledMethod) {
```

Comment:
(Optional) I don't think we normally call functions sendXXXMessage so I would just call it `revealInModelEditor`.

Or maybe even just roll it into `viewInModelEditor` instead of having calling another function, since it's so simple (warning: browser coding).

```ts
const viewInModelEditor = useCallback(
  () => vscode.postMessage({
    t: "revealInModelEditor",
    modelAlerts.model,
  });
}, [modelAlerts.model]);
```